data/nav_graphql.yml

```
@@ -3,8 +3,22 @@
   path: apis/graphql-api
 - name: Console and CLI tutorial
   path: apis/graphql/graphql-tutorial
-- name: Cookbook
-  path: apis/graphql/graphql-cookbook
+- name: Cookbooks
+  children:
+  - name: Pipelines
```

Based on my other comment, something like:

```suggestion
- name: Cookbook
  children:
  - name: Overview
    path: apis/graphql/graphql-cookbook
  - name: Pipelines
```